install/gcp/installer/util/utils.py

```diff
@@ -232,37 +262,34 @@ def infer_version(advanced_mode):
     Returns:
         str: Selected Forseti branch.
     """
-    return_code, out, err = run_command(
-        ['git', 'rev-parse', '--abbrev-ref', 'HEAD'])
-    if return_code:
-        print(err)
-        print('Will try to infer the Forseti version instead.')
-    else:
-        branch = out.strip()
 
-    user_choice = None
-    if not branch or branch == 'master':
-        version = get_forseti_version()
-        if version:
-            branch = 'v%s' % version
+    cur_version = get_forseti_version()
```

Comment:
Should we add a check here, if not advanced mode:
```
if not advanced_mode:
    cur_version = get_forseti_version()
    if not cur_version:
        sys.exit(1)
    return cur_version
````

Reply:
If it's advanced_mode we still want to use the get_forseti_version() to get the default version and show it to the user.

Reply:
Then can we do this, return right away if not in advanced mode?

```
cur_version = get_forseti_version()
if not cur_version:
    sys.exit(1)

if not advanced_mode:
    return cur_version
```